ros_msft_onnx/include/ros_msft_onnx/yolo_box.h

```diff
@@ -22,6 +22,22 @@ namespace yolo
     class YoloProcessor : public OnnxProcessor
     {
         std::string _label;
+        std::string _label_file_path;
+        std::vector<std::string> _labels{
+        "aeroplane", "bicycle", "bird", "boat", "bottle",
```

Comment:
I'd like to get this out of this file and into a config

Reply:
To add more context, the tags themselves are specific to the test data model - this should not be in the code.

Reply:
You can add this to the launch file for the test data - use a string array.